BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1601,28 +1555,69 @@ IEnumerator FireReloadableMissile()
                     ml.LoftVelComp = LoftVelComp;
                     ml.LoftVertVelComp = LoftVertVelComp;
                     //ml.LoftAltComp = LoftAltComp;
-                    ml.pronavGain = pronavGain;
                     ml.loftState = LoftStates.Boost;
                     ml.TimeToImpact = float.PositiveInfinity;
                 }
-                if (homingModeTerminal == GuidanceModes.APN || homingModeTerminal == GuidanceModes.PN)
-                    ml.pronavGain = pronavGain;
+                /*if (GuidanceMode == GuidanceModes.AAMHybrid)
+                    ml.pronavGain = pronavGain;*/
 
-                if (homingModeTerminal == GuidanceModes.Kappa)
+                if (GuidanceMode == GuidanceModes.Kappa)
                 {
                     ml.kappaAngle = kappaAngle;
                     ml.LoftAngle = LoftAngle;
-                    ml.loftState = LoftStates.Boost;
                     ml.LoftTermAngle = LoftTermAngle;
                     ml.LoftMaxAltitude = LoftMaxAltitude;
                     ml.LoftRangeFac = LoftRangeFac;
                     ml.LoftVertVelComp = LoftVertVelComp;
                     ml.LoftRangeOverride = LoftRangeOverride;
+                    ml.loftState = LoftStates.Boost;
+                }
+            }
+            
+            if (terminalHoming)
+            {
+                if (homingModeTerminal == GuidanceModes.AGMBallistic)
+                {
+                    ml.BallisticOverShootFactor = BallisticOverShootFactor; //are some of these null, and causeing this to quit? 
+                    ml.BallisticAngle = BallisticAngle;
+                }
+                if (homingModeTerminal == GuidanceModes.Cruise)
+                {
+                    ml.CruiseAltitude = CruiseAltitude;
+                    ml.CruiseSpeed = CruiseSpeed;
+                    ml.CruisePredictionTime = CruisePredictionTime;
                 }
 
-                ml.terminalHomingRange = terminalHomingRange;
-                ml.homingModeTerminal = homingModeTerminal;
-                ml.terminalHomingActive = false;
+                if (BDArmorySettings.DEBUG_MISSILES)
```

Comment:
Some of the loft fields are being set active in the editor (e.g., `LoftMaxAltitude` from line 1017), but they're not persistent now.  That might be somewhat confusing for users.

Reply:
They're active but the sliders are not active, they're meant to serve as reference for the user in the editor, since I felt putting all of that info into the part info box would overly clutter it.